test/Integration/test-csharp/GetProductColumnTypesSerialization.cs

```diff
@@ -27,13 +23,8 @@ public static class GetProductsColumnTypesSerialization
             [Sql("SELECT * FROM [dbo].[ProductsColumnTypes]",
                 CommandType = System.Data.CommandType.Text,
                 ConnectionStringSetting = "SqlConnectionString")]
-            IAsyncEnumerable<ProductColumnTypes> products,
-            ILogger log)
+            IEnumerable<ProductColumnTypes> products)
```

Comment:
We should be consistent and call Serialize on the items in the IEnumerable as well